pkg/api/v1/atlasproject_types.go

```diff
@@ -200,6 +204,19 @@ func (p *AtlasProject) ConnectionSecretObjectKey() *client.ObjectKey {
 	return nil
 }
 
+func (p *AtlasProject) BackupCompliancePolicyObjectKey() *client.ObjectKey {
```

Comment:
how does this differ from `project.Spec.BackupCompliancePolicyRef.GetObject(project.Namespace)`?

Reply:
removed and replaced with `GetObject`